MyLibrary/MyLibraryApp.Application/Models/Books/DeleteBookRequestModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MyLibraryApp.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.Application.Models.Books
+{
+    public class DeleteBookRequestModel
+    {
+        [Required]
+        public int Id { get; set; }
+
+        [Required]
+        public string Title { get; set; }
```

Comment:
When deleting a book you only need it id. The rest of the properties are useless and can be safely removed.